src/parsers/mama-mia.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const moment = require('moment');
+
+function parseData(html, date) {
+    const json = JSON.parse(html);
+    const days = ["Pondělí", "Úterý", "Středa", "Čtvrtek", "Pátek"];
+    const today = date.day() - 1
+    let menu = {
+        found: false
+    };
+    if (json.data) {
+        for (let i = 0; i < 7; i++) {
+            const text = json.data[i].message;
+            const attachments = json.data[i].attachments;
+            if (text != undefined &&
+                text.toLowerCase().includes(days[today].toLowerCase()) &&
+                attachments != undefined) {
```

Comment:
`text !=undefined && attachments != undefined && ...` are not common in js,
rather `text && attachments && ...`